project/settings/defaults.py

```diff
@@ -31,6 +31,10 @@
 AUTH_USER_MODEL = 'accounts.User'
 GUARDIAN_MONKEY_PATCH = False
 
+# S3 Bucket name where video attachments are stored
+BUCKET_NAME = os.environ.get('BUCKET_NAME', 'fakeBucketName')
```

Comment:
@pattisdr Just for future reference, settings that will be different per environment should be expected to be in each environment and we should provide a default that either a) works on local dev or b) pretty clearly spells out that it's fake.